src/project/internal/saveprojectscenario.cpp

```diff
@@ -422,12 +426,26 @@ Ret SaveProjectScenario::showCloudSaveError(const Ret& ret, const CloudProjectIn
     int defaultButtonCode = okBtn.btn;
 
     switch (ret.code()) {
```

Comment:
It looks like this duplicates ProjectActionsController::showScoreDownloadError. It would be great to have this code in only one place

Reply:
I've extracted some common logic from these methods by moving `showScoreDownloadError` to ~SaveProjectScenario~OpenSaveProjectScenario.